src/matrix_t/matrix_t.h

```diff
@@ -36,7 +36,7 @@ class matrix_t {
       // printf("type is : %i\n", type);
       // u8.assign(size - 1, 0);
       std::cout << "size parameter: " << size << std::endl;
-      for (int i = 0; i < size - 1; i++) {
```

Comment:
we are pushing values in the vector until the limit of the size. So it shouldn't `size -1` but only `size`